thread/unix/rasthrsup.c

```diff
@@ -27,50 +27,19 @@
 #include "omrthread.h"
 
 #if defined(LINUX)
-#if __GLIBC_PREREQ(2,4)
 #include <sys/syscall.h>
-#endif /* __GLIBC_PREREQ(2,4) */
 #elif defined(OSX)
 #include <pthread.h>
 #include <sys/syscall.h>
 #endif /* defined(LINUX) */
 
```

Comment:
1. `<pthread.h>` is already included above, at line 23, so the code for OSX and linux can be the same
2. there is a comment at line 26, "this #include defines the buildspec flags". I would rather explicitly include omrcfg.h at the top of the list.

The includes should look something like this:

```c++
#include "omrcfg.h"

#include <pthread.h>
#if defined(LINUX) || defined(OSX)
#include <sys/syscall.h>
#endif /* defined(LINUX) || defined(OSX) */
#include <unistd.h>

#include "omrthread.h"
```